app/models/classification.rb

```diff
@@ -149,6 +147,24 @@ def to_s
     name
   end
 
+  def policy_content_ids
+    classification_policies.map(&:policy_content_id)
+  end
+
+  def policy_content_ids=(content_ids)
+    self.classification_policies = content_ids.map do |content_id|
+      ClassificationPolicy.new(policy_content_id: content_id)
+    end
+  end
+
+  def policies
+    Future::Policy.from_content_ids(policy_content_ids)
+  end
+
+  def can_be_related_to_policies?
```

Comment:
I don't believe this method is required - it's an edition thing that gets overridden in the `Edition::RelatedPolicies` mixin.